### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by delishad21**
Just some small changes to the ReadMe file in our repository.

### CODE REVIEW

**Review by Zer0Legion — COMMENTED** [review]
Great change to prepare for our final release.

File: README.md [hunk 1]
```diff
@@ -3,14 +3,21 @@
 
 
 ## Avengers Assemble
-Avengers Assemble is a contact management app, meant for use with a Command Line Interface (CLI) while still maintaining the benefits of a Graphical User Interface (GUI).
 
-The application is designed for Head Tutors of the CS1101S Programming Methodology course but its use cases can be extended to Head Tutors of other courses.
+![Ui](docs/images/Ui.png)
+
+Avengers Assemble is a contact management app, meant for use with a Command Line Interface (CLI) while still maintaining the benefits of a Graphical User Interface (GUI).
```
Comment (jayllo-c) [inline README.md:10]: Neat changes

**Review by jayllo-c — COMMENTED** [review]
Makes the readme more professional

File: README.md [hunk 2]
```diff
@@ -3,14 +3,22 @@
...
+* [User Guide](https://ay2324s2-cs2103t-t10-1.github.io/tp/UserGuide.html)
+* [Developer Guide](https://ay2324s2-cs2103t-t10-1.github.io/tp/DeveloperGuide.html)
+* [About Us](https://ay2324s2-cs2103t-t10-1.github.io/tp/AboutUs.html)
+
+
+### Acknowledgements
+
+* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
+* Libraries used: [JavaFX](https://openjfx.io/), [Jackson](https://github.com/FasterXML/jackson), [JUnit5](https://github.com/junit-team/junit5), [OpenCSV](https://opencsv.sourceforge.net/)
+
```
Comment (danielleloh) [inline README.md:24]: all changes look good to me :)

**Review by danielleloh — COMMENTED** [review]
nice edit to readme :)

**Pull request merged**